### ISSUE TITLE
possible to see npm output on bootstrap

### ISSUE THREAD

**Issue by tyler-johnson** [lerna/lerna#1601]
Would it be possible to see NPM output when bootstrapping? It's mostly the deprecation and peer dependencies notices that would be nice to see when installing. The only time I seem to get them currently is when NPM has an error while building.

Maybe this is already possible and I missed a flag somewhere?

**Comment by evocateur** [comment]
No, it's not currently possible, mostly because I haven't personally used the bootstrap command in almost a year. PRs welcome.
